doc/operators/ledger_snapshot.rst

```diff
@@ -0,0 +1,90 @@
+Ledger and Snapshots Management
+===============================
+
+The ledger and snapshot files written by CCF nodes to disk should be managed by operators to allow for safe backup of the service and application state as well as fast join and recovery procedures. This section describes how these files are generated and how operators should manage them effectively.
+
+.. note:: See the :doc:`/audit/index` section to read about offline ledger auditability.
+
+Ledger
+------
+
+The ledger is the persistent replicated append-only record of the transactions that have been executed by the CCF service. It is written by the primary node when a transaction is executed and replicated to all backups which maintain their own duplicated copy. Each node in a network creates and maintains its own local copy of the ledger. Committed entries are always identical between :ref:`a majority <design/consensus:CFT Consensus Protocol>` of nodes, but a node may be more or less up to date, and uncommitted entries may differ.
```

Comment:
We're a little vague about transactions here, and whether that's reads/writes. We could/should perhaps elaborate for the BFT and CFT cases.